src/ui/src/app/repositories/repositories.component.scss

```diff
@@ -0,0 +1,104 @@
+@import '../../theme.scss';
+$filters-width: 175px;
+
+.repositories {
+  &__header {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+
+    &__title {
+      margin-top: 0;
+    }
+
+    &__filters {
+      display: none;
+      font-size: 0.5em;
+
+      md-icon {
+        margin-left: 10px;
+        top: 6px;
+        position: relative;
+      }
+    }
+
+    &__new {
+      margin-left: 1em;
+      margin-bottom: 1em;
+      text-align: center;
+    }
+
+  }
+  // CSS from https://codepen.io/AllThingsSmitty/pen/MyqmdM
+  table {
+    border: 1px solid #ccc;
```

Comment:
Please, use the colors defined in `theme.scss` or a variation of them (`lighten`, `darken`...)